components/input/index.jsx

```diff
@@ -108,6 +108,10 @@ class Input extends React.Component {
 			label: PropTypes.string,
 			spinner: PropTypes.string,
 		}),
+		/**
+		 * Disabled brower's autocomplete when "off" is used.
+		 */
+		autoComplete: PropTypes.string,
```

Comment:
Should I add a new test just for this prop ?

Reply:
All props that can be changed either need to be tracked via snapshot tests or mocha tests.

Reply:
I think autocomplete='test' is in one of the snap shots.

Reply:
Oh, I'm sorry. I misread this. I thought it was in combobox. Yes, a snapshot is fine.